cmd/gcContent/gcContent.go

```diff
@@ -0,0 +1,96 @@
+// Command Group: "FASTA and Multi-FASTA Tools"
+
+// Calculates the gc content of fasta sequences for all regions specified in a bed file
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/fileio"
+	"log"
+)
+
+func gcContent(bedFile string, faFile string, outFile string, multiFaMode bool, species string) {
+
+	var currRecordSeq []dna.Base
+	var found bool
+	var gc float64
+	var annotated bed.Bed
+	var speciesStart, speciesEnd int
+
+	regionsChan := bed.GoReadToChan(bedFile)
+	records := fasta.Read(faFile)
+	recordsMap := fasta.ToMap(records)
+	out := fileio.EasyCreate(outFile)
+
+	for curr := range regionsChan {
+
+		if multiFaMode {
+			// when multiFa, each fasta sequence is a species, e.g. >human, >hca, assuming that the user specified the correct chromosome, e.g. this multi-fasta is chr1.fa
+			currRecordSeq, found = recordsMap[species]
```

Comment:
So if in multiFa mode, you need to provide a species name? If that is true, I would mention that in the usage, and also have a check in main() to see if multiFaMode was set without a species being specified.